requirements.txt

```
@@ -2,15 +2,15 @@
 # as ranges in order to support interoperability with other mkdocs plugins or
 # packages that might otherwise exist in an adopter's environment.
 mkdocs>=1.6
-Markdown>=3.2,<3.4
+Markdown>=3.2,<3.8
 
 # The following are more akin to direct dependencies. Each line represents one
 # or more features that are provided by `techdocs-core`, and thus are always
 # pinned to an exact version. Bumps should be accompanied by release notes
 # explaining what was added or fixed (or at least pointing to the underlying
 # release notes of the bumped package).
 mkdocs-material==9.5.42
-markdown_inline_graphviz_extension==1.1.2
+markdown_inline_graphviz_extension@git+https://github.com/cesaremorel/markdown-inline-graphviz.git@v1.1.3
```

```suggestion
markdown_inline_graphviz_extension===1.1.3
```

Is there a reason we don't just do the above?

---

Because they never published to pypi for this version.

---

```suggestion
markdown_inline_graphviz_extension@git+https://github.com/cesaremorel/markdown-inline-graphviz.git@579f10af9fe7187c717c20615f65774f898c1a0d
```

Is it possible to do this using a commit hash just to provider a bit more security here?